recipe/meta.yaml

```diff
@@ -39,6 +41,7 @@ test:
         - fiona.fio
     commands:
         - fio --help
+        - conda inspect linkages fiona --name _test  # [linux]
```

Comment:
> `(e.g. assert_links $SP_DIR/fiona/ogrext* libgif* $PREFIX)`

We should work on something like that. It would be a nice addition to our tests. I will try to come up with something.